### DISCUSSION TITLE
Select Time zone

### DISCUSSION THREAD

**Discussion by ValouNova** [time4tea/gopro-dashboard-overlay#174] [Ideas]
Hello,
I use this program to make an overlay from a gopro video and with the data taken by the gopro.
But I always try to get the UTC time on my overlay.
I changed the gopro time to UTC, but the overlay remains UTC-5, my computer is UTC-5 and I'm in France so UTC+1.
I've noticed that the time zone used is that of the computer, but would it be possible to determine ourselves in which time zone the video was taken, so that the program faces the correct change to UTC?

Thanks in advance.

Val

**Comment by time4tea** [comment]
Hiya.. depending on how you run the programme it should be possible to change the timezone to what you want.

Of course the normal use case is that you are in the same timezone as you did the recording, and the computer is also set in that timezone.

If you want to change the timezone, and you're using Linux or mac, you should be able to say:

`TZ=Europe/London venv/bin/gopro-dashboard.py ....`  and it will use that timezone.

There are a number of standard timezone names, so you might want `TZ=Europe/Paris`

The gopro itself always uses GPS time, which is  the same (for practical purposes) as UTC.

**Reply by ValouNova** [reply]
Thank you very much for your prompt reply.
I'm on windows and I couldn't find the TZ variable in the gopro-dashboard.py file. Does it have an equivalent in a Windows script?
Yes, the camera is in UTC, but the reference used is always that of the computer apparently (according to my observations of course).

Thank you very much for your help and for your work.

**Reply by time4tea** [reply]
I've been doing a bit of reading, and on Windows this is much harder... I might have to add a flag to the program so it is easier.

Its a system wide setting on windows, so more tricky to change.

Thanks for bringing it to my attention.
